engine/src/main/java/org/terasology/engine/entitySystem/event/AbstractValueModifiableEvent.java

```diff
@@ -71,9 +71,7 @@ public float getResultValue() {
      * Calculates the result value from the base value and given modifiers and multipliers.
      * <p>
      * The value is calculated based on the following formula:
-     * <pre>
-     * result = (<baseValue> + Σ <modifier>) * Π <multiplier> + Σ <postModifier>
-     * </pre>
```

Comment:
I'm not sure about this change - what was the problem here? 

`{@code ...}` will be an in-line verbatim text, like `this`. However, `<pre>...</pre>` should create a block of "preformatted" text, like
``` 
this
``` 

I intentionally put this formula into a block to be rendered separately.

Reply:
`pre` blocks don't like `<` and `>`, that's the problem